plugin.video.jpcandioti.5rtv/main.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Module: plugin.video.jpcandioti.5rtv
+# Author: Juan Pablo Candioti - @JPCandioti
+# Site: https://github.com/jpcandioti/plugin.video.jpcandioti.5rtv
+# Created on: 2020-03-21
+
+import sys
+import xbmcgui
+import xbmcplugin
+
+_HANDLE = int(sys.argv[1])
+
+LIVESTREAM_CHANNEL  = '22636012'
+LIVESTREAM_EVENT    = '8242619'
+YOUTUBE_CHANNEL     = 'UCCvR_NFSKLkPM-7zbGV2Ckg'
+
+
+def mainlist():
+    listing = []
+
+    list_item = xbmcgui.ListItem(label='En vivo')
```

Comment:
please add `En vivo` to localized strings (strings.po)

Reply:
I fixed it. Thanks!